zb-db/src/main/java/io/zeebe/db/ZeebeDb.java

```diff
@@ -61,4 +62,11 @@
    * @return {@code true} if the column is empty, otherwise {@code false}
    */
   boolean isEmpty(ColumnFamilyType column, DbContext context);
+
+  <KeyType extends DbKey, ValueType extends DbValue> void visit(
+      final ColumnFamilyType column,
+      final DbContext context,
+      final KeyType keyInstance,
+      final ValueType valueInstance,
+      final BiConsumer<KeyType, ValueType> visitor);
```

Comment:
Please add some documentation and mention that this method should be used for tests only.

(optional) We could rename the method to `forEach()` to make the intent more clear.